domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131800__project_seed_data

```diff
@@ -22,10 +22,10 @@ db.project.insertMany([
     "tenantId": "PB",
     "code": "PR3",
     "name": "Project3",
-    "eatId": "EAT3",
-    "departmentId": "DEP3",
-    "locationId": [
-      "L111","L121"
+    "expenditureId": "a4c21fc1-28df-494b-8983-baf3c6b027d7",
+    "departmentId": "3d9ef18a-361a-40cf-b142-dd6f998e1ad3",
```

Comment:
I assume these departmentId and expenditureId exist in their corresponding seed data. Can you confirm it?

Reply:
yes